### PR TITLE
Added Discord support.

### PR DESCRIPTION
**Pull request by alon710**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,13 +15,25 @@ CREDENTIALS_ID=<user identification number>
...
+
 TELEGRAM_BOT_TOKEN=<telegram bot token>
 TELEGRAM_CHAT_ID=<telegram chat id>
 
-BUDGET_ILS=<budget in ILS>
+DISCORD_BOT_TOKEN=<discord bot token>
+DISCORD_CHANNEL_ID=<discord channel id>
+```
+
+2. Supported Destinatons
```
Comment (elongl) [inline README.md]: This should be at the top, outside of the _Setup_ section.

File: README.md [hunk 2]
```diff
@@ -15,13 +15,25 @@ CREDENTIALS_ID=<user identification number>
 CREDENTIALS_CARD_DIGITS=<last 6 digits of card>
 CREDENTIALS_PASSWORD=<password>
 
+BUDGET_ILS=<budget in ILS>
+
+DESTINATIONS=<comma deperated list if uppercase destinations>
```
Comment (elongl) [inline README.md:27]:
```suggestion
DESTINATIONS=<comma deperated list of uppercase destinations>
```

File: index.js [hunk 3]
```diff
@@ -49,21 +50,25 @@ const getTotalCharged = (scrapeResult) => {
...
-  const bot = new TelegramBot(process.env.TELEGRAM_BOT_TOKEN);
-  const chatId = process.env.TELEGRAM_CHAT_ID;
-  bot.sendMessage(chatId, message);
-};
-
 const main = async () => {
   const budget = parseBudget();
   const scrapeResult = await scrape();
   const totalCharged = getTotalCharged(scrapeResult);
+  const destination = process.env.DESTINATIONS.split(",");
```
Comment (elongl) [inline index.js]: This should be named `destinations` (plural).

File: index.js [hunk 4]
```diff
@@ -1,6 +1,7 @@
 import { CompanyTypes, createScraper } from "israeli-bank-scrapers";
+import { sendTelegramMessage } from "./telegram.js";
```
Comment (elongl) [inline index.js]: You didn't include those modules.

Comment (alon710) [reply index.js]: What do you mean?

Comment (elongl) [reply index.js]: There's no `telegram.js` and `discord.js` in the PR.

File: index.js [hunk 5]
```diff
@@ -46,24 +47,29 @@ const getTotalCharged = (scrapeResult) => {
...
+  return Math.round(totalCharged * 100) / 100;
 };
 
 const main = async () => {
   const budget = parseBudget();
   const scrapeResult = await scrape();
   const totalCharged = getTotalCharged(scrapeResult);
+  const destinations = process.env.DESTINATIONS.split(",");
+  let message;
+  const persantage = Math.round((totalCharged / budget) * 100);
```
Comment (elongl) [inline index.js]: percentage*

File: index.js [hunk 6]
```diff
@@ -46,24 +47,29 @@ const getTotalCharged = (scrapeResult) => {
...
     });
   });
-  return totalCharged;
-};
-
-const sendTelegramMessage = (message) => {
-  const bot = new TelegramBot(process.env.TELEGRAM_BOT_TOKEN);
-  const chatId = process.env.TELEGRAM_CHAT_ID;
-  bot.sendMessage(chatId, message);
+  return Math.round(totalCharged * 100) / 100;
```
Comment (elongl) [inline index.js:50]: There's no need for that if you've already done the rounding afterwards.

File: index.js (hunk 5 above)
Comment (elongl) [reply index.js]: Also call that `exceed_percent`.

File: index.js (hunk 6 above)
Comment (alon710) [reply index.js:50]: This is the rounding for the price, the other is for the `exceed_percent`

**Pull request merged**